docs/running-on-mesos.md

```diff
@@ -516,6 +516,16 @@ See the [configuration page](configuration.html) for information on Spark config
     Fetcher Cache</a>
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.checkpoint</code></td>
+  <td>false</td>
+  <td>
+    If set, agents running tasks started by this framework will write the framework pid, executor pids and status updates to disk. 
```

Comment:
Let's customize this copy a bit for Spark instead of just copying the protobuf docs.  e.g. "tasks" should be "executors" and you should remove the part about "this framework", in place of something about Spark.